site/content/3.12/develop/http-api/collections.md

```diff
@@ -950,6 +950,45 @@ logJsonResponse(response);
 db._drop(cn);
 ```
 
+### Get the available key generators
+
+```openapi
+paths:
+  /_api/key-generators:
+    get:
+      operationId: getKeyGenerators
+      description: |
+        Returns the available key generators for collections.
+      responses:
+        '200':
+          description: |
+            A list of the available generators for document keys:
+            - `"padded"`
+            - `"uuid"`
+            - `"autoincrement"`
+            - `"traditional"`
+          content:
+            application/json:
+              schema:
+                type: array
```

Comment:
This is actually incorrect (my bad), it's an object at the top-level, and an array under the `keyGenerators` property. The toolchain basically doesn't seem to support arrays, which is a problem here but also for a few other endpoints (and more in the future if we move from full-text to structural descriptions)